docs/content/config/index.md

```diff
@@ -74,7 +74,7 @@ more secure authentication like 2FA or WebAuthn.
 | `HD_AUTH_LOCAL_ENABLE_REGISTER`           | `false` | `true`, `false`         | This makes it possible to register new local accounts in HedgeDoc.                    |
 | `HD_AUTH_LOCAL_MINIMAL_PASSWORD_STRENGTH` | `2`     | `0`, `1`, `2`, `3`, `4` | The minimum [zxcvbn-ts][zxcvbn-ts-score] password score, that passwords need to have. |
 
-**Password score ([zxcvbn-ts][zxcvbn-ts-score])**
+#### Password score ([zxcvbn-ts][zxcvbn-ts-score])
```

Comment:
The links shouldn't be present in the headline

Reply:
What would you recommend instead? A short sentence beneath?
`The password score is calculated with [zxcvbn-ts][zxcvbn-ts-score].`

Reply:
sounds good